src/main/modules/oidc/index.ts

```diff
@@ -34,7 +34,7 @@ export class OidcMiddleware {
 
     app.use(
       errorHandler(async (req: AppRequest, res: Response, next: NextFunction) => {
-        if (req.session?.user && req.session.user.roles.includes('citizen')) {
+        if (req.session?.user.roles.includes('citizen')) {
```

Comment:
Better to have the optional chaining for user variable too?
Ex: req.session?.user?.roles.includes('citizen')